test/object-omit/index.js

```diff
@@ -35,6 +36,7 @@ test('omit using arguments', function(t) {
 test('omit using a non-existent key', function(t) {
   t.plan(1);
   var obj = {a: 3, b: 5, c: 9};
+  // @ts-ignore
```

Comment:
The next line is ignored, because it intentionally passes a key name that is not found in `obj` above.